### PR TITLE
Fixed javaScript bug for multiple different select2 fields on one page

### PR DESCRIPTION
**Pull request by anneFly**

### CODE REVIEW

File: django_select2/static/django_select2/django_select2.js [hunk 1]
```diff
@@ -27,16 +26,19 @@
...
     };
 
     $.fn.djangoSelect2 = function (options) {
         var settings = $.extend({}, options);
-        var heavy = $(this).hasClass('django-select2-heavy');
-        if (heavy) {
-            return initHeavy(this, settings);
-        }
-        return init(this, settings);
+        $.each(this, function (i, el) {
```
Comment (codingjoe) [inline django_select2/static/django_select2/django_select2.js]: please use better variable names `el` should be `element`

File: django_select2/static/django_select2/django_select2.js [hunk 2]
```diff
@@ -27,16 +26,19 @@
         }, options);
 
         $el.select2(settings);
```
Comment (codingjoe) [inline django_select2/static/django_select2/django_select2.js]: why not call init at this point?

**Comment by codingjoe** [comment]
@anneFly I need to you write a simple selenium test this time. Just write a form with two different fields.
That should be fairly simple.

**Comment by anneFly** [comment]
@codingjoe there was already a view in the testapp, that had multiple fields on one page, I just extended it so that every select2 field shows different data. Is that okay?
Tests looked fine to me, but I'm not able to run those tests that require phantomjs, yet. Gotta build it from source on linux and it's taking ages. Once I make it run, I can try to write the selenium test (if you're not faster)

**Comment by anneFly** [comment]
@codingjoe made it finally and added a selenium test

**Comment by codingjoe** [comment]
Thanks @anneFly great work.

Released in 5.3.1

**Pull request closed without merging**